package.json

```diff
@@ -9,19 +9,19 @@
     "express": "^4.14.1"
   },
   "devDependencies": {
-    "codecov": "^2.1.0",
+    "codecov": "^3.6.1",
     "compression": "^1.6.2",
     "cors": "^2.8.1",
     "eslint": "^3.14.1",
```

Comment:
https://github.com/CloudNativeJS/appmetrics-prometheus/network/alerts also flags eslint:
![image](https://user-images.githubusercontent.com/5445507/67976755-9de0ae00-fc0e-11e9-8796-417b8b10aa6b.png)

Reply:
Force pushed this in, upgraded to v4.18.2 and `eslint` still seems to pass.